lib/docs/scrapers/tensorflow/tensorflow.rb

```diff
@@ -14,24 +14,14 @@ class Tensorflow < UrlScraper
     options[:container] = '.devsite-main-content'
 
     options[:attribution] = <<-HTML
-      &copy; 2019 The TensorFlow Authors. All rights reserved.<br>
+      &copy; 2020 The TensorFlow Authors. All rights reserved.<br>
       Licensed under the Creative Commons Attribution License 3.0.<br>
       Code samples licensed under the Apache 2.0 License.
     HTML
 
-    version '2.1' do
-      self.release = '2.1.0'
-      self.base_url = "https://www.tensorflow.org/versions/r#{version}/api_docs/python"
-    end
-
-    version '2.0' do
-      self.release = '2.0.0'
-      self.base_url = "https://www.tensorflow.org/versions/r#{version}/api_docs/python"
-    end
-
-    version '1.15' do
-      self.release = '1.15.0'
-      self.base_url = "https://www.tensorflow.org/versions/r#{version}/api_docs/python"
+    for version in ['2.3', '2.2', '2.1', '2.0', '1.15'] do
```

Comment:
The statement `version '2.3' do … end` invokes the method `version` from `lib/docs/core/doc.rb` and cannot be replaced by a loop like this. I've fixed this syntax